.github/workflows/deploy.yml

```diff
@@ -0,0 +1,13 @@
+on:
+  workflow_call:
```

Comment:
This Github Action runs when we call it from other actions, nice for making re-usable Github Actions workflows

This is the workflow that will auto-deploy all our changes to Fly.io